packages/blade/docs/tokens/Colors.stories.mdx

```diff
@@ -0,0 +1,109 @@
+import { Meta, DocsContainer, Story, ColorPalette, ColorItem } from '@storybook/addon-docs';
```

Comment:
A lot of formatting issues on this file. Can you check locally? It should use prettier here as well